srfi-test/tests/srfi/14/srfi-14.rkt

```diff
@@ -67,7 +67,8 @@
 ;; As of Unicode 7.0: 104077
 ;; As of Unicode 14.0: 133396
 ;; As of Unicode 15.0: 137765
-(test 137765 char-set-size char-set:letter)
+;; As of Unicode 15.1: 138387			  
+(test 138387 char-set-size char-set:letter)
```

Comment:
This number is what the failing test's "actual" value tells us, but is it the actually the right number? From https://unicode.org/versions/Unicode15.1.0/:

> Unicode 15.1 adds 627 characters, for a total of 149,813 characters. 

Also, note that the difference between 15.1 and 15.0 is 138387 - 137765 = 622, which is not 627.

Reply:
OK, I missed that it's counting `char-set:letter`. 

Is there a reference where we can find `(char-set-size char-set:letter)` to compare against? Otherwise, we rely on actually running the code, which makes the test less useful.

Reply:
The number 622 appears here: https://www.unicode.org/charts/PDF/Unicode-15.1/

Reply:
That's not useful though. There are many blocks, and one needs to know if a block is a part of `char-set:letter` or not, and sums across potentially many blocks to compute that number. It just happens that for 15.1, "622" happens to be there.

Reply:
This change looks correct to me. The change to Unicode source tables can be seen as part of racket/racket@c1b908cd0ad21b6aa1afc91b78fddd0ca7ba934c (a long as you trust the old and new versions to be the real files). In particular, "PropList.txt" shows the addition of one block of 622 "Lo"s. The other additions are in categories that do not start with "L".

Reply:
Sorry for this nitpicking. Let me start by saying that my comment should not prevent the PR from being merged (and in fact, let me just merge it now).

My point though, is that it would be nice to find an easy-to-find, verifiable quantity to test against. The test here (`(char-set-size char-set:letter)` evaluates to `138387`) is probably not wrong, but it's difficult to tell that is the case unless you know how to read `PropList.txt`, and even if you do, it's still difficult to extract the number.

My proposal: pages like https://unicode.org/versions/Unicode15.1.0/ contains:

> Unicode 15.1 adds 627 characters, for a total of 149,813 characters.

Can we find an expression that evaluates to "149813"? This "149813" is what I meant by "easy-to-find, verifiable quantity". I don't need to sum things up to arrive at the number.

I experimented with `(char-set-size char-set:full)`, but that returns `1112064`, and I don't understand how this number could exceed "149813".

Reply:
I am not aware of a place to find totals like that other than to add up ranges from `PropList.txt`.

`1112064` is the number of values for which `integer->char` returns a value. That's the length of the range `#x0` to `#x10FFFF` inclusive minus the length of the range used for surrogate pairs, `#xD800` to `#xDFFF` inclusive.